iXBRLViewerPlugin/viewer/src/js/viewer.js

```diff
@@ -194,6 +198,13 @@ Viewer.prototype.elementForFact = function (fact) {
     return $('.ixbrl-element', this._contents).filter(function () { return $(this).data('ivid') == fact.id }).first();
 }
 
+Viewer.prototype.elementsForFacts = function (facts) {
+    var ids = $.map(facts, function (f) { return f.id });
+    var elements = $('.ixbrl-element', this._contents).filter(function () { return $.inArray($(this).data('ivid'), ids ) > -1 });
+    console.log(elements);
```

Comment:
Should maybe remove this log statement, and/or limit the output

Reply:
Remove is probably the best option, if we host this on a server, we don't want it logging pre-release data.

Reply:
Removed in da396ed